daffodil-test/src/test/resources/org/apache/daffodil/extensions/doubleToRawLong/doubleToRawLong.tdml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<tdml:testSuite xmlns:ex="http://example.com" xmlns="http://example.com"
+  xmlns:tdml="http://www.ibm.com/xmlns/dfdl/testData" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/" xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  xmlns:dfdlx="http://www.ogf.org/dfdl/dfdl-1.0/extensions"
+  xmlns:fn="http://www.w3.org/2005/xpath-functions"  >
+
+  <tdml:defineSchema name="doubleToRawLong.dfdl.xsd">
+
+    <xs:include schemaLocation="org/apache/daffodil/xsd/DFDLGeneralFormat.dfdl.xsd"/>
+    <dfdl:format ref="ex:GeneralFormat"
+      lengthUnits="bytes" lengthKind="explicit" length="8"
+      dfdlx:emptyElementParsePolicy="treatAsEmpty"
+      representation="binary"
+
+      />
+
+   <xs:element name="root">
+     <xs:complexType>
+       <xs:sequence>
+       <xs:element type="xs:double" name="DoubleValue"  dfdl:length="8"   />
+        <xs:element type="xs:long" name="LongValue" dfdl:inputValueCalc="{ dfdlx:doubleToRawLong(../ex:DoubleValue) }" />
+       </xs:sequence>
+     </xs:complexType>
+   </xs:element>
+
+  </tdml:defineSchema>
+  <tdml:parserTestCase name="doubleToRawLong"
+    root="root" model="doubleToRawLong.dfdl.xsd" description="Extensions - doubleToRawLong">
+    <tdml:document>
+   <tdml:documentPart type="byte">
+     5DC8 225C AF53 6E1F
+    </tdml:documentPart>
+    </tdml:document>
+
+    <tdml:infoset>
+      <tdml:dfdlInfoset xmlns:xs="http://www.w3.org/2001/XMLSchema"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+        <root>
+        <DoubleValue>5.886014640420331E143</DoubleValue>
+        <LongValue>6757689022343245343</LongValue>
+        </root>
+      </tdml:dfdlInfoset>
+    </tdml:infoset>
+  </tdml:parserTestCase>
+
+</tdml:testSuite>
```

Comment:
The indentation is inconsisent in this file. Suggest fixing it up or running it through a formatter.